core/server/src/main/java/alluxio/RestUtils.java

```diff
@@ -36,7 +61,10 @@ public static Response createResponse() {
    * @param object the object to respond with
    * @return the response
    */
-  public static Response createResponse(Object object) {
+  private static Response createResponse(Object object) {
+    if (object instanceof Void) {
+      return Response.ok().build();
+    }
```

Comment:
Will this behave correctly if we called something not of type Void but it happens to return null?